packages/flutter/lib/src/material/theme.dart

```diff
@@ -161,6 +161,48 @@ class Theme extends StatelessWidget {
         .resolveFrom(context);
   }
 
+  /// Retrieves the [Brightness] to use for descendant widgets, based
```

Comment:
I would keep the mention of `Material`, since `Theme` is a `Material` thing and not under the widgets library.

```suggestion
  /// Retrieves the [Brightness] to use for descendant Material widgets, based
```